talkative_robot_2.rb

```diff
@@ -0,0 +1,307 @@
+require 'pry'
```

Comment:
Keep the file names the same. If not, I can't track changes and have to read through every line. :(